charts/snowplow-iglu-server/values.yaml

```diff
@@ -13,7 +13,7 @@ service:
   port: 8080
   image:
     repository: "snowplow/iglu-server"
-    tag: "0.8.7"
+    tag: "0.8.7-distroless"
```

Comment:
@jbeemster I wonder what you think about this line.

I thought it would be great if we support only the distroless image.  Not the non-distroless image.  So users who don't know the difference get the more secure image automatically.

To do this, I would need to change `tag` to something like `version: 0.8.7`, and then we add the `-distroless` suffix somewhere else.  But then it would look a bit strange to have `repository` as a configurable thing, but not the tag.  So I didn't do it.

Reply:
I think this is nice for the moment to just control it via the tag - when we push the new helm chart out users will get the version we want them to use anyway.

Is there a future where distroless just becomes the defacto standard image as well?

Reply:
If we make it the defacto standard immediately, I suspect many open source users will find it break their current deployment process.  Because we know they are not just using the image as we say to; they are also modifying the image in strange ways.

If we make it easier to supply application config to our apps, then open source users will have less reason to modify the image.  And then maybe we can move 100% to distroless.

Changes like:
- All apps must accept a path to a config file.  See e.g. snowplow/snowplow-rdb-loader#966
- Iglu resolver config file should resolve environment variables.  I started to work on this in snowplow/enrich#731